hyprshot

```diff
@@ -146,6 +169,10 @@ function begin_grab() {
                 local geometry=`grab_window`
             fi
             ;;
+        all)
+            save_all
+            return
+            ;;
```

Comment:
I would prefer if you followed the convention of the other modes. Maybe so that you don't need to calculate the geometry in this instance, you can check if geometry was not passed in to `save_geometry` and just call grim without any geometry.